connector_prestashop/product.py

```diff
@@ -382,38 +456,35 @@ def _get_quantity(self, record):
             all_qty += int(quantity['quantity'])
         return all_qty
 
-    def _get_product(self, record):
+    def _get_template(self, record):
         if record['id_product_attribute'] == '0':
-            binder = self.get_binder_for_model('prestashop.product.product')
-            return binder.to_openerp(record['id_product'], unwrap=True)
-        binder = self.get_binder_for_model('prestashop.product.combination')
-        return binder.to_openerp(record['id_product_attribute'], unwrap=True)
+            binder = self.binder_for('prestashop.product.template')
+            return binder.to_odoo(record['id_product'], unwrap=True)
+        binder = self.binder_for('prestashop.product.combination')
+        return binder.to_odoo(record['id_product_attribute'], unwrap=True)
```

Comment:
No, the binder for `prestashop.product.combination` returns a `product.product`:

```python
class PrestashopProductCombination(models.Model):
    _name = 'prestashop.product.combination'
    _inherit = 'prestashop.binding'
    _inherits = {'product.product': 'openerp_id'}

    openerp_id = fields.Many2one(
        comodel_name='product.product',
        string='Product',
        required=True,
        ondelete='cascade',
    )
```